common/app/common/Navigation.scala

```diff
@@ -128,7 +129,8 @@ trait Navigation {
 
   //Technology
   val technologyblog = SectionLink("technology", "technology blog", "Technology blog", "/technology/blog")
-  val games = SectionLink("culture", "games", "Games", "/technology/games")
+  val gamesLink = if(GamesLinkSwitch.isSwitchedOn) "/games" else "/technology/games"
```

Comment:
private? or you can directly do `val games = SectionLink("culture", "games", "Games", if(GamesLinkSwitch.isSwitchedOn) "/games" else "/technology/games")`